webpack.config.js

```diff
@@ -38,6 +38,8 @@ module.exports = {
 		...coblocksChunks,
 		'coblocks-extensions': path.resolve( process.cwd(), 'src/extensions.js' ),
 		'coblocks-plugin-deactivation': path.resolve( process.cwd(), 'src/components/coblocks-deactivate-modal/index.js' ),
+		'coblocks-styles': path.resolve( process.cwd(), 'src/block-styles.js' ),
+		'coblocks-variations': path.resolve( process.cwd(), 'src/block-variations.js' ),
 
```

Comment:
I wonder now if we should name this differently for posterity. These are programmatic 'style' extensions as opposed to CSS style-related code but the naming might confuse. Maybe these should instead say 
`coblocks-styles-extension` and `coblocks-variations-extension` for clarity. What do you think @jrtashjian?